_posts/2020-05-14-status-update.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "14st May 2020: IAC Team Status Update"
+author: daianamezdrea
+categories:
+  - team
+  - status
+tags:
+  - module updates
+  - community
+  - litmus
+  - testing
+  - dsc
+  - pdk
+  - powershell
+  - windows
+  - modulebuilder
+  - voxpupuli
+---
+The IAC team has been working on a lot of interesting things over the past week. We want to share with our community some pretty interesting things. Here's a brief overview of what we've been up to:
+
+## COVID-19 who?
+We don't count anymore the days as a fully remote working team, but we count the days when we work hard and with all the energy we have for the best results!
+`#unstoppableteam #workHard #staysafe #thinkpositive`
+
+## Module Releases
+
+Modules released this week:
+- [puppetlabs/apt](https://github.com/puppetlabs/puppetlabs-apt) (7.4.2)
+
+## PDK 1.18.0.0 was released
+
+Thanks to [scotje](https://github.com/scotje) for his help in getting this release out the door:
+- [puppetlabs/pdk](https://github.com/puppetlabs/pdk) (1.18.0)
+
+Big thanks for this releases to our team members too: [Ciaran](https://github.com/sanfrancrisko), [David](https://github.com/david22swan), [Paula](https://github.com/pmcmaw) and [Sheena](https://github.com/sheenaajay).
+
+## Community Contributions
+
+We think that is no doubt about our community contributors, they are always ready to help and they have excellent ideas. Here are some highlights of this week:
+
+Thank you to [akerl-unpriv](https://github.com/akerl-unpriv) for his contribution to the [puppetlabs-firewall](https://github.com/puppetlabs-firewall) module. We now have support for the `cgroup` flag in iptables.
+Thanks to [crazymind1337](https://github.com/crazymind1337) for [the great effort](https://github.com/puppetlabs/puppetlabs-docker/pull/607) cleaning up the `puppetlabs-docker` repo. It’s fantastic to see the open source community committed to keeping the standard high.
+
+##Paula continues her way back to Puppet goodies
+
+This is what she said when she was working on puppetlabs-apt release:
```

Comment:
Please add a link to the release tooling here. Hopefully folx will want to see what paula is raving about here :-D